src/isValidNumber.js

```diff
@@ -0,0 +1,26 @@
+import { either } from 'ramda';
+
+import isFloat from './isFloat';
+import isInteger from './isInteger';
+
+/* eslint-disable max-len */
+/**
+ * Checks if value is valid `Number` object and not `NaN`,
```

Comment:
Checks if value is a valid `Number`. A valid `Number` is a number that is not `NaN`, `Infinity` or `-Infinity`.